src/styles/main.scss

```diff
@@ -1,3 +1,32 @@
-body {
-  margin: 0;
+@import "./variables";
+@import "./normalize";
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax($sizeBlock, 1fr));
+}
+
+@media (min-width: 900px) {
+  .container {
+    grid-template-columns: repeat(3, minmax($sizeBlock, 1fr));
```

Comment:
not sure if min-max is needed here

Reply:
I added it because of this rule. `The width of each block should be not less than its height (300px)`